back/lib/shared/vault/vault_test.go

```diff
@@ -3,12 +3,41 @@ package vault_test
 import (
 	"testing"
 
-	. "github.com/onsi/ginkgo"
-	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
-func TestVaultAPI(t *testing.T) {
+func TestVaultClient(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecsWithDefaultAndCustomReporters(t, "vault", []Reporter{reporters.NewJUnitReporter("test_report_vault.xml")})
+	RunSpecs(t, "VaultClient")
 }
+
+/*
+
+var _ = Describe("ReadSecret", func() {
+	Context("When we want to read a secret on HCP Vault Secrets", func() {
+		payload := `{"secret":{"name":"hello_world_secret", "version":{"version":"1", "type":"kv", "created_at":"2024-01-15T21:48:33.292865Z", "value":"hello_world", "created_by":{"name":"john.doe@gmail.com", "type":"TYPE_USER", "email":"john.doe@gmail.com"}}, "created_at":"2024-01-15T21:48:33.292865Z", "latest_version":"1", "created_by":{"name":"john.doe@gmail.com", "type":"TYPE_USER", "email":"john.doe@gmail.com"}, "sync_status":{}}}`
+		httpClient := &httpclient.HttpClientMock{Resp: []httpclient.MockResponse{
+			{Data: []byte(payload), Status: 200}, {Data: []byte(payload), Status: 200},
+		}}
+
+		vlt, err := vault.InitializeVaultApi(httpClient, vault.Vault{
+			AppName:        "Krispy",
+			AuthURL:        "blabalbla",
+			BaseURL:        "BASE",
+			OrganizationID: "123",
+			ProjectID:      "456",
+		})
+
+		It("should not return an error during initialization", func() {
+			Expect(err).To(Not(HaveOccurred()))
+		})
+
+		secret, err := vlt.ReadSecret("hello_world_secret")
+		It("should return the secret value without error", func() {
+			Expect(err).To(Not(HaveOccurred()))
+			Expect(secret).To(Equal("hello_world"))
+		})
+	})
+})
+*/
```

Comment:
Shouldn't this code be removed?

Reply:
It's going to be used later on, I'll remove it for now. THanks